hedera-node/hedera-app-spi/src/main/java/com/hedera/node/app/spi/state/ReadableKVState.java

```diff
@@ -84,4 +84,11 @@ default boolean contains(@NonNull final K key) {
      */
     @NonNull
     Set<K> readKeys();
+
+    /**
+     * Gets the size of the {@link ReadableKVState} backing store.
+     * @return size of the data source.
+     */
+    @NonNull
+    long size();
```

Comment:
This seems wrong. I think the size should be the size as it is understood by the `ReadableKVState` and `WritableKVState`. In the case of the `ReadableKVState`, this is indeed the size of the backing store. But in the case of the `WritableKVState`, it is the size of the backing store +/- any modifications (additions and removals).

Reply:
Makes sense. Makes sense. Changed to include modifications.

Reply:
Makes sense. Makes sense. Changed to include modifications. Thank you !